packages/palette/.storybook/preview.js

```diff
@@ -1,40 +1,59 @@
-import React from "react"
-import { addDecorator } from "@storybook/react"
+import React, { useState } from "react"
 import { Theme } from "../src/Theme"
+import { THEMES } from "../src/themes"
 import { injectGlobalStyles } from "../src/helpers/injectGlobalStyles"
 import { INITIAL_VIEWPORTS } from "@storybook/addon-viewport"
 import { breakpoints } from "../src/Theme"
+import { Tabs, Tab } from "../src/elements/Tabs"
 
 const { GlobalStyles } = injectGlobalStyles()
 
-addDecorator((storyFn) => (
-  <Theme>
-    <>
-      <GlobalStyles />
-      {storyFn()}
-    </>
-  </Theme>
-))
-
-const viewports = Object.entries(breakpoints).reduce(
-  (memo, [key, width]) => {
-    if (['__docgenInfo', 'displayName'].includes(key)) {
-      return memo
-    }
-
-    return {
-      ...memo,
-      [key]: {
-        name: `breakpoint: ${key}`,
-        styles: {
-          width,
-          height: "100%",
-        },
-      },
-    }
+export const decorators = [
+  (Story) => {
+    const [theme, setTheme] = useState(localStorage.getItem("theme") ?? "v2")
+
+    return (
+      <Theme theme={THEMES[theme]}>
+        <>
+          <GlobalStyles />
+
+          <Tabs
+            initialTabIndex={["v2", "v3"].indexOf(theme)}
+            onChange={({ name }) => {
+              setTheme(name)
+              localStorage.setItem("theme", name)
+            }}
+          >
+            <Tab name="v2">
+              <Story />
+            </Tab>
+
+            <Tab name="v3">
+              <Story />
+            </Tab>
+          </Tabs>
+        </>
+      </Theme>
+    )
```

Comment:
This is the decorator which loads the tabs for every story and swaps out the theme. I'm persisting the active tab in local storage so that they stay active between stories.